src/config.ts

```diff
@@ -447,17 +448,40 @@ export const getClientProfileRedirectionUrl = (
     href: url
   };
 };
+
+export const ClientErrorRedirectionUrlParams = t.union([
+  t.intersection([
+    t.interface({
+      errorMessage: NonEmptyString
+    }),
+    t.partial({
+      errorCode: t.number
+    })
+  ]),
+  t.intersection([
+    t.partial({
+      errorMessage: NonEmptyString
+    }),
+    t.interface({
+      errorCode: t.number
+    })
+  ]),
```

Comment:
```suggestion

```
I think we could remove this block.

Reply:
Why? If we remove this block we cannot call this
```ts
this.getClientErrorRedirectionUrl({
  errorCode: AGE_LIMIT_ERROR_CODE
});
```

Reply:
you are right, I missed it.